examples/common/tracing/decoder/logging/Log.cpp

```diff
@@ -76,9 +76,19 @@ void Log(const char * name, ByteSpan & data)
         return;
     }
 
-    char buffer[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE];
+    // If the size of the data is larger than half of the maximum size for a log message (minus 1 for null-termination),
+    // reduce the size of the data to fit within this limit.
```

Comment:
```suggestion
    // reduce the size of the data to fit within this limit.  The limit is half the size of the message
    // because we will be logging a hex representation of that data, at 2 chars per byte.
```